core/server/data/migrations/versions/4.35/2022-01-30-15-17-set-welcome-page-url-from-settings.js

```diff
@@ -0,0 +1,39 @@
+const logging = require('@tryghost/logging');
+
+// For DML - data changes
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Setting welcome_page_url from settings');
+
+        const paidSignupRedirect = await knex()
```

Comment:
@allouis I think knex expects the format of `await knex('settings')...`

```
[2022-01-31 14:23:23] INFO Setting welcome_page_url from settings
calling knex without a tableName is deprecated. Use knex.queryBuilder() instead.
calling knex without a tableName is deprecated. Use knex.queryBuilder() instead.
```

Reply:
Done!